tests/test-runner.cpp

```diff
@@ -67,6 +67,10 @@ int main(int argc, char** argv)
 
   edge_init(&enclave);
 
+  enclave.measure(argv[1], argv[2], params);
```

Comment:
@alexthomas1 Can you add another utility to just measure the enclave and exit?
(instead of measuring the enclave in test-runner.cpp)

Reply:
You might want to create `tools` directory in sdk,
add a utility (with whatever name you think right), and a Makefile
so that running `make` will just build a separate utility that only measures the enclave